### PR TITLE
BIM: Fix buildingsmart-tech.org url links

### PR DESCRIPTION
**Pull request by 3x380V**
See #14253

### CODE REVIEW

File: src/Mod/BIM/bimcommands/BimPreflight.py [hunk 1]
```diff
@@ -894,8 +894,8 @@ def testPsets(self):
                     "\n"
                     + translate(
                         "BIM",
-                        "Verify which properties a certain property set must contain on http://www.buildingsmart-tech.org/ifc/IFC4/Add2/html/annex/annex-b/alphabeticalorder_psets.htm",
-                    )
+                        "Verify which properties a certain property set must contain on %1",
+                    ).replace("%1", "https://standards.buildingsmart.org/IFC/DEV/IFC4_2/FINAL/HTML/annex/annex-b/alphabeticalorder_psets.htm")
```
[the remark this answers is not in the document]
Comment (3x380V) [reply src/Mod/BIM/bimcommands/BimPreflight.py]: Good suggestion, thank you. I will open separate PR for this issue with above included. After all it was a bit of this PR topic.

File: src/Mod/BIM/bimcommands/BimPreflight.py [hunk 2]
```diff
@@ -321,8 +321,8 @@ def testIFC4(self):
                 msg = (
                     translate(
                         "BIM",
-                        "ifcopenshell is not installed on your system or not available to FreeCAD. This library is responsible for IFC support in FreeCAD, and therefore IFC support is currently disabled. Check https://www.freecadweb.org/wiki/Extra_python_modules#IfcOpenShell to obtain more information.",
-                    )
+                        "ifcopenshell is not installed on your system or not available to FreeCAD. This library is responsible for IFC support in FreeCAD, and therefore IFC support is currently disabled. Check %1 to obtain more information.",
+                    ).replace("%1", "https://www.freecadweb.org/wiki/Extra_python_modules#IfcOpenShell")
```
[the remark this answers is not in the document]
Comment (3x380V) [reply src/Mod/BIM/bimcommands/BimPreflight.py]: Thank you, see comment bellow.

**Comment by 3x380V** [comment]
Changes suggested by @furgo16 are now part of their own #14301

**Comment by yorikvanhavre** [comment]
> Changes suggested by @furgo16 are now part of their own https://github.com/FreeCAD/FreeCAD/pull/14301

This PR should still be merged, right?

**Comment by 3x380V** [comment]
@yorikvanhavre, yes. Draft status will vanish once actually runtime tested.

**Pull request merged**